lib/collection/view/third_pass_help_page.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+
+class ThirdPassHelpPage extends StatelessWidget {
+  const ThirdPassHelpPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(title: const Text('Third Pass Help')),
+        body: Column(children: const [
+          Padding(
+            padding: EdgeInsets.all(8.0),
+            // TODO: refactor into something nicer
+            // TODO: make it scrollable to prevent RenderFlex Bottom Overflow
+            child: Text(
+                'Please start with the first food item recorded in first pass.'
```

Comment:
Same sentiment as before - perhaps a pop up implementation of sorts might be more useful to prompt the enumerator

Reply:
Hmm, I personally think a pop up is almost the same as a new page (assuming the popup covers most of the screen), but maybe some other implementation could be considered. Anyways, we can ask ICRISAT about it later.